tokio-util/src/io/inspect.rs

```diff
@@ -0,0 +1,115 @@
+use futures_core::ready;
+use pin_project_lite::pin_project;
+use std::{
+    io::{IoSlice, Result},
+    pin::Pin,
+    task::{Context, Poll},
+};
```

Comment:
We don't use multi-line imports in Tokio.
```suggestion
use std::io::{IoSlice, Result};
use std::pin::Pin;
use std::task::{Context, Poll};
```

Reply:
Does this style correspond to [Module](https://rust-lang.github.io/rustfmt/?version=v1.5.1&search=imports#Module%5C%3A) in rustfmt? If so, should I put up a pull request for adding a `rustfmt.toml` that sets this as config?

Reply:
Yes, we use the module style. A PR sounds good.

Reply:
This needs nightly rustfmt due to [Module being unstable](https://github.com/rust-lang/rustfmt/issues/4991) and changes formatting of quite a few files. I'll put up the PR anyway, but I'm expecting it to be rejected as too disruptive.